src/cirrus/git_tools.py

```diff
@@ -159,10 +156,26 @@ def commit_files(repo_dir, commit_msg, *filenames):
     # commits with message
     new_commit = repo.index.commit(commit_msg)
     # push branch to origin
-    result = repo.remotes.origin.push(repo.head)
+    result = None
+    if push:
+        result = repo.remotes.origin.push(repo.head)
     return result
```

Comment:
You could reorganize this, as by default the function will return None:
```
if push:
    return repo.remotes.origin.push(repo.head)
```

No need for the None assignment or return in the case `push` is `True`